palera1n.sh

```diff
@@ -596,52 +610,40 @@ if [ ! -f blobs/"$deviceid"-"$version".shsh2 ]; then
         fi
     fi
 
-    if [ -z "$no_install" ]; then
-        tipsdir=$(remote_cmd "/usr/bin/find /mnt2/containers/Bundle/Application/ -name 'Tips.app'" 2> /dev/null)
-        sleep 1
-        if [ "$tipsdir" = "" ]; then
-            echo "[!] Tips is not installed. Once your device reboots, install Tips from the App Store and retry"
-            remote_cmd "/sbin/reboot"
-            sleep 1
-            _kill_if_running iproxy
-            exit
-        fi
-        remote_cmd "/bin/mkdir -p /mnt1/private/var/root/temp"
-        sleep 1
-        remote_cmd "/bin/cp -r /usr/local/bin/loader.app/* /mnt1/private/var/root/temp"
-        sleep 1
-        remote_cmd "/bin/rm -rf /mnt1/private/var/root/temp/Info.plist /mnt1/private/var/root/temp/Base.lproj /mnt1/private/var/root/temp/PkgInfo"
-        sleep 1
-        remote_cmd "/bin/cp -rf /mnt1/private/var/root/temp/* $tipsdir"
-        sleep 1
-        remote_cmd "/bin/rm -rf /mnt1/private/var/root/temp"
-        sleep 1
-        remote_cmd "/usr/sbin/chown 33 $tipsdir/Tips"
-        sleep 1
-        remote_cmd "/bin/chmod 755 $tipsdir/Tips $tipsdir/PogoHelper"
-        sleep 1
-        remote_cmd "/usr/sbin/chown 0 $tipsdir/PogoHelper"
-    fi
-
     #remote_cmd "/usr/sbin/nvram allow-root-hash-mismatch=1"
     #remote_cmd "/usr/sbin/nvram root-live-fs=1"
-    if [ "$semi_tethered" = "1" ]; then
-        remote_cmd "/usr/sbin/nvram auto-boot=true"
+    if [ "$tweaks" = "1" ]; then
+        if [ "$semi_tethered" = "1" ]; then
+            remote_cmd "/usr/sbin/nvram auto-boot=true"
+        else
+            remote_cmd "/usr/sbin/nvram auto-boot=false"
+        fi
     else
-        remote_cmd "/usr/sbin/nvram auto-boot=false"
+        remote_cmd "/usr/sbin/nvram auto-boot=true"
     fi
 
-    remote_cp binaries/Kernel15Patcher.ios root@localhost:/mnt1/private/var/root/Kernel15Patcher.ios
-    remote_cmd "/usr/sbin/chown 0 /mnt1/private/var/root/Kernel15Patcher.ios"
-    remote_cmd "/bin/chmod 755 /mnt1/private/var/root/Kernel15Patcher.ios"
-
     # lets actually patch the kernel
     echo "[*] Patching the kernel"
-    remote_cmd "rm -f /mnt6/$active/System/Library/Caches/com.apple.kernelcaches/kcache.raw /mnt6/$active/System/Library/Caches/com.apple.kernelcaches/kcache.patched /mnt6/$active/System/Library/Caches/com.apple.kernelcaches/kcache.im4p /mnt6/$active/System/Library/Caches/com.apple.kernelcaches/kernelcachd"
-    if [ "$semi_tethered" = "1" ]; then
-        remote_cmd "cp /mnt6/$active/System/Library/Caches/com.apple.kernelcaches/kernelcache /mnt6/$active/System/Library/Caches/com.apple.kernelcaches/kernelcache.bak"
+    remote_cmd "rm -f /mnt1/private/var/root/kpf"
+    if [[ "$version" == *"16"* ]]; then
+        if [ "$semi_tethered" = "1" ]; then
+            remote_cp binaries/Kernel16Patcher-nolivefs.ios root@localhost:/mnt1/private/var/root/kpf
```

Comment:
I guess someone does not know how to parse arguments

Reply:
skull emoji

Reply:
Where can I find this file Kernel16Patcher-nolivefs.ios?
I need this file to try on iOS 16, thanks a lot

Reply:
on  god